travis/.travis.yml

```diff
@@ -6,7 +6,8 @@ cache:
 before_install:
   - "script/clone_all_rspec_repos"
   # Note this doesn't work on JRUBY 2.0.0 mode so we don't do it, the excluded versions are broken on Ruby 2.3
-  - if [ "jruby" != "$TRAVIS_RUBY_VERSION"  ]; then gem install bundler --version "!= 1.12.0, != 1.12.1"; fi
+  - if [ "jruby" != "$TRAVIS_RUBY_VERSION"  ]; then gem install bundler --version "!= 1.12.0, != 1.12.1, != 1.12.2, != 1.12.3"; fi
+install: if [ "jruby" != "$TRAVIS_RUBY_VERSION"  ]; then bundle _1.11.2_ install --binstubs --standalone --without documentation --path ../bundle; else bundle install --binstubs --standalone --without documentation --path ../bundle; fi
```

Comment:
Also, instead of putting the whole `bundle` command, you can just alias `bundle` to `bundle _1.11.2_`.  See 35ac13ff79673cfeb61ec9145fae0018f021804c for an example.